postreview/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+from django.utils import reverse
+
+User = get_user_model()
+
+
+class review(models.Model):

```

Comment:
The name of the class should begin with a capital letter. Please follow the naming conventions.